Assertion triggered on LAB diagram export

A diagram in my model seems to trigger an assertion in the rendering code.
The obfuscated model is attached ([bf12dada.zip](https://github.com/user-attachments/files/17378425/bf12dada.zip))

Relevant Versions:
capellambse == 0.6.7
Python == 3.12.4
Environment: Windows

Steps to reproduce:
```python
import capellambse
path_to_model = "bf12dada.aird"
model = capellambse.MelodyModel(path_to_model)
diagram = model.diagrams.by_name("dceef58b-6584-45fe-8514-5ccf6024bccadc")
diagram.save("dceef58b-6584-45fe-8514-5ccf6024bccadc.png", "png")
```

I am not aware of anything special about the diagram.

Expected output
![dceef58b-6584-45fe-8514-5ccf6024bccadc](https://github.com/user-attachments/assets/d5c9fc1c-ad39-47d1-94fc-45dbc2bf4002)
Results in:

```text.python.traceback
AssertionError                            Traceback (most recent call last)
Cell In[6], line 5
      3 model = capellambse.MelodyModel(path_to_model)
      4 diagram = model.diagrams.by_name("dceef58b-6584-45fe-8514-5ccf6024bccadc")
----> 5 diagram.save("dceef58b-6584-45fe-8514-5ccf6024bccadc.png", "png")

File [C:\stuff\capella\pycapella-env\Lib\site-packages\capellambse\model\diagram.py:451](file:///C:/stuff/capella/pycapella-env/Lib/site-packages/capellambse/model/diagram.py#line=450), in AbstractDiagram.save(self, file, fmt, pretty_print, **params)
    446     else:
    447         raise ValueError(
    448             f"No known extension for format {fmt!r},"
    449             " specify a file name explicitly"
    450         )
--> 451 data = self.render(fmt, pretty_print=pretty_print, **params)
    452 if isinstance(data, str):
    453     data = data.encode("utf-8")

File [C:\stuff\capella\pycapella-env\Lib\site-packages\capellambse\model\diagram.py:406](file:///C:/stuff/capella/pycapella-env/Lib/site-packages/capellambse/model/diagram.py#line=405), in AbstractDiagram.render(self, fmt, pretty_print, **params)
    403         if not self._allow_render:
    404             raise RuntimeError(f"Diagram not in cache: {self.name}")
--> 406 render = self.__render_fresh(params)
    407 return _run_converter_chain(chain, render, pretty_print=pretty_print)

File [C:\stuff\capella\pycapella-env\Lib\site-packages\capellambse\model\diagram.py:577](file:///C:/stuff/capella/pycapella-env/Lib/site-packages/capellambse/model/diagram.py#line=576), in AbstractDiagram.__render_fresh(self, params)
    574         self._render = self.__create_error_image("parse", self._error)
    576 if hasattr(self, "_error"):
--> 577     raise self._error
    578 return self._render

File [C:\stuff\capella\pycapella-env\Lib\site-packages\capellambse\model\diagram.py:571](file:///C:/stuff/capella/pycapella-env/Lib/site-packages/capellambse/model/diagram.py#line=570), in AbstractDiagram.__render_fresh(self, params)
    569 self.invalidate_cache()
    570 try:
--> 571     self._render = self._create_diagram(params)
    572 except Exception as err:
    573     self._error = err

File [C:\stuff\capella\pycapella-env\Lib\site-packages\capellambse\model\diagram.py:723](file:///C:/stuff/capella/pycapella-env/Lib/site-packages/capellambse/model/diagram.py#line=722), in Diagram._create_diagram(self, params)
    722 def _create_diagram(self, params: dict[str, t.Any]) -> diagram.Diagram:
--> 723     return aird.parse_diagram(self._model._loader, self._element, **params)

File [C:\stuff\capella\pycapella-env\Lib\site-packages\capellambse\aird\__init__.py:228](file:///C:/stuff/capella/pycapella-env/Lib/site-packages/capellambse/aird/__init__.py#line=227), in parse_diagram(model, descriptor, **params)
    226 for data_elm in treedata.iterdescendants("children", "edges"):
    227     try:
--> 228         elm = _element_from_xml(
    229             C.ElementBuilder(
    230                 target_diagram=diag,
    231                 diagram_tree=dgtree,
    232                 data_element=data_elm,
    233                 melodyloader=model,
    234                 fragment=diag_descriptor.fragment,
    235             )
    236         )
    237     except C.SkipObject:
    238         continue

File [C:\stuff\capella\pycapella-env\Lib\site-packages\capellambse\aird\__init__.py:263](file:///C:/stuff/capella/pycapella-env/Lib/site-packages/capellambse/aird/__init__.py#line=262), in _element_from_xml(ebd)
    261 else:
    262     factory = _visual.from_xml
--> 263 return factory(ebd)

File [C:\stuff\capella\pycapella-env\Lib\site-packages\capellambse\aird\_semantic.py:98](file:///C:/stuff/capella/pycapella-env/Lib/site-packages/capellambse/aird/_semantic.py#line=97), in from_xml(ebd)
     86     raise c.SkipObject()
     88 seb = c.SemanticElementBuilder(
     89     target_diagram=ebd.target_diagram,
     90     diagram_tree=ebd.diagram_tree,
   (...)
     96     melodyobjs=melodyobjs,
     97 )
---> 98 return drawtype(seb)

File [C:\stuff\capella\pycapella-env\Lib\site-packages\capellambse\aird\_semantic.py:126](file:///C:/stuff/capella/pycapella-env/Lib/site-packages/capellambse/aird/_semantic.py#line=125), in FactorySelector.__call__(self, seb)
    124 else:
    125     raise ValueError(f"Unknown element type {seb.data_element.tag}")
--> 126 return factory(seb)

File [C:\stuff\capella\pycapella-env\Lib\site-packages\capellambse\aird\_edge_factories.py:69](file:///C:/stuff/capella/pycapella-env/Lib/site-packages/capellambse/aird/_edge_factories.py#line=68), in generic_factory(seb)
     67     snaptarget(edge, -1, -2, targetport, not edge.hidden, routingstyle)
     68 if isinstance(sourceport, diagram.Box):
---> 69     snaptarget(edge, 0, 1, sourceport, not edge.hidden, routingstyle)
     71 edge.labels.extend(_construct_labels(edge, seb))
     73 sourceport.add_context(seb.data_element.attrib["element"])

File [C:\stuff\capella\pycapella-env\Lib\site-packages\capellambse\aird\_edge_factories.py:304](file:///C:/stuff/capella/pycapella-env/Lib/site-packages/capellambse/aird/_edge_factories.py#line=303), in snaptarget(***failed resolving arguments***)
    302     snap_tree(points, i, next_i, target)
    303 else:
--> 304     snap_oblique(points, i, next_i, target)

File [C:\stuff\capella\pycapella-env\Lib\site-packages\capellambse\aird\_edge_factories.py:315](file:///C:/stuff/capella/pycapella-env/Lib/site-packages/capellambse/aird/_edge_factories.py#line=314), in snap_oblique(points, i, next_i, target)
    313 """Snap ``points``' end to ``target`` in a straight line."""
    314 source = points[next_i]
--> 315 new_point = target.vector_snap(
    316     points[i], style=diagram.RoutingStyle.OBLIQUE, source=source
    317 )
    318 delta = (points[i] - source).angleto(new_point - source)
    319 if abs(delta) >= 1:

File [C:\stuff\capella\pycapella-env\Lib\site-packages\capellambse\diagram\_diagram.py:278](file:///C:/stuff/capella/pycapella-env/Lib/site-packages/capellambse/diagram/_diagram.py#line=277), in Box.vector_snap(self, point, source, style)
    276     if point == source:
    277         return self.__vector_snap_closest(point)
--> 278     return self.__vector_snap_oblique(point, source)
    279 if style is RoutingStyle.MANHATTAN:
    280     return self.__vector_snap_manhattan(point, point - source)

File [C:\stuff\capella\pycapella-env\Lib\site-packages\capellambse\diagram\_diagram.py:368](file:///C:/stuff/capella/pycapella-env/Lib/site-packages/capellambse/diagram/_diagram.py#line=367), in Box.__vector_snap_oblique(self, point, source)
    365             intersections.append(intersection)
    367 assert len(intersections) > 0, f"{edge} doesn't intersect {edges}"
--> 368 assert len(intersections) < 2, f"{edge} intersects multiple {edges}"
    369 return intersections[0]

AssertionError: (Vector2D(x=681.3416631630917, y=177.0), Vector2D(x=806.3218673218673, y=368.0)) intersects multiple {'top', 'left'}
```

--- a/capellambse/diagram/_diagram.py
+++ b/capellambse/diagram/_diagram.py
@@ -365,7 +365,6 @@ class Box:
                     intersections.append(intersection)
 
         assert len(intersections) > 0, f"{edge} doesn't intersect {edges}"
-        assert len(intersections) < 2, f"{edge} intersects multiple {edges}"
         return intersections[0]
 
     def __vector_snap_manhattan(

